tests/common.go

```diff
@@ -1228,6 +1234,36 @@ func ValidateMountOptionsWithPureVolumes(ctx *scheduler.Context, errChan ...*cha
 
 }
 
+// ValidateCreateOptionsWithPureVolumes is the ginkgo spec for executing file system options check for the given volume
+func ValidateCreateOptionsWithPureVolumes(ctx *scheduler.Context, errChan ...*chan error) {
+	var requiredXfsCreateoption = "-b 2048"
+	var requiredExt4Createoption = "-b size=2048"
+	vols, err := Inst().S.GetVolumes(ctx)
+	log.FailOnError(err, "Failed to get app %s's volumes", ctx.App.Key)
+	log.Infof("volumes of app %s are %s", ctx.App.Key, vols)
+	for _, v := range vols {
+		pvcObj, err := k8sCore.GetPersistentVolumeClaim(v.Name, v.Namespace)
+		log.FailOnError(err, " Failed to get pvc for volume %s", v)
```

Comment:
Please make this func return error and then handle the error in the test

Reply:
Now we are handling the error here with processError when error is returned from this function.

Reply:
Is there any particular reason you want to process and handle all errors for this particular func with `processError()` and `log.FailOnError()` instead of just returning it and handling it inside the test? I see a lot of similar functions return errors and this one doesn't, I am just trying to understand why not have this function also return error instead of handling it inside

Reply:
I have added this test for createoption check in already existing FA/FB test `PureVolumeCRUDWithSDK` in which this is a step and most of those tests handle the error inside the function only when they are getting called in steps of function `ValidateContextForPureVolumesSDK`. For example you can look at function `ValidatePureSnapshotsSDK` which is getting called in one of the steps and it handles its error inside the function only.